internal/ir/xds.go

```diff
@@ -262,6 +264,22 @@ func (t TLSListenerConfig) Validate() error {
 	return errs
 }
 
+type PathEscapedSlashAction egv1a1.PathEscapedSlashAction
```

Comment:
why not using `egv1a1.PathEscapedSlashAction` directly?

Reply:
Because if I use it directly, I would need to import `egv1a1` into the XDS layer in order to translate the API constants into XDS constants.